### DISCUSSION TITLE
"Journaling" with freeplane ?

### DISCUSSION THREAD

**Discussion by bepolymathe** [freeplane/freeplane#476] [General]
Hi, 

I would like to set up a journaling system to keep track of my daily work and thoughts. Have any of you set up something like this?
I'm looking for ideas! 🤔 
Thanks for your help.

**Comment by macmarrum** [comment]
Hi @bepolymathe 

I recommend looking at

* [Some examples](./254)
* [Plan-and-organize add-ons](https://docs.freeplane.org/#/getting-started/Add-ons_(install)?id=plan-and-organize)
* [newJournalNode.groovy](https://github.com/macmarrum/freeplane-scripting/blob/main/src/scripts/newJournalNode.groovy) – a script that automates creating new journal nodes (remove the first line for the script to appear in `Tools->Scripts` instead of `Mac2`)

**Comment by bepolymathe** [comment]
Thanks @macmarrum !
I will try it in the next days.
